minutes/2023-10-17.md

```diff
@@ -0,0 +1,246 @@
+        * David: observing - SCI wg working on overall model - toolbelt 
+            * Crob: 7 groups working on threatmodels - i think it should be streamlined
+            * David: it should be a group wide - merging various views into one view.
+        * Ryan: Individual groups MVSRs should have traceability up to org’s MVSR…
+        * Arnaud: we had an effort to get every wg a charter - but we didn’t give people proper directives …  Don’t know if we need MVSR for each group but we need something like a scope statement.
+        * Omkhar: I don't want to get rid of charters or MVSRs - maybe one M, one S, one V for the whole org without each group needing its own MVSR. I have no opinion, just let’s agree on what we’re doing.
+* Review new work - (seeking TAC comments, edits, approvals) - ISSUES & PRs filed in TAC Repo
+    * [TAC Issue 208](https://github.com/ossf/tac/issues/208) - WG Lead R&R - _10min_
+    * We saw there was an ill-defined set of roles & responsibilities for a wg lead… we’re trying to correct that.
+        * Isaac: great list - making sure that the WGs are “legible” to other working groups is a key responsibility. E.g. every time we have a meeting, post a little summary in the slack - so we can see there what we’ve discussed.
+        * Dan: 2-3 hours, not 1-2… 
+        * Ryan: agree with both the above points.
+        * Crob: we would share this with WG leads and get their feedback and then publish this as part of the body of documentation within the TAC. Then we’ll document the roles & responsibilities of working in the TAC. Please add comments to the issue.
+* Question about where we store the meeting recordings.
+    * Amanda: in your “personal meeting spot” in LFX.: [https://openprofile.dev/my-calendar](https://openprofile.dev/my-calendar) 
+* Updates on older matters 
+    * **INFORM **- [TAC PR 178](https://github.com/ossf/tac/pull/178) - Guac move to Incubating
+    * **INFORM **- With the upcoming Operational Model discussions with the GB, the TAC will be updating our TI definitions, lifecycle stages, and associated documentation within [https://github.com/ossf/tac/tree/main/process](https://github.com/ossf/tac/tree/main/process) 
+    * 
+        * TIs will be _Working Groups, Projects_ (expanded now to cover specs), and _SIGs_
+        * All TIs will share a common lifecycle (that will be tailored for each specific category as needed) - _Sandbox, incubating, Graduated, Archived_
+        * 
+
+<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image7.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image7.png "image_tooltip")
```

Comment:
The image will need to be added later (@hythloda said that wouldn't be a problem)